CAToolsetPerfDataCollector/README.md

```diff
@@ -0,0 +1,96 @@
+    <paths to trace files> : Paths to one or more ETW trace files, separated by space. Required.
+Option names and values are case-insensitive.
+```
+
+By default is will print CA performance summary report. If "-f:CSV" option is specified, it will print per translation unit performance data in CSV format for each and every translation unit that is built and analyzed. If "-f:both" option is specified, it will print both CA performance summary report and per translation unit performance data.
+
+## Example Outputs
+
+### CA Performance Summary Report
+This is a example output of CA performance data summary report. Please note that the actual execution times and their relative percentage numbers will vary project by project, as well as CA configurations.
+
+Percentage number for each child is ralative to its immediate parent. So, sum of the percentage numbers for all of the siblings should always equal to 100%.
+
+```
+Number of TUs successfully analyzed: 3289 
+Number of TUs excluded (files not analyzed or had analysis error): 1893 
+Total Execution Time (milliseconds): 
+  Front End Pass, BackEnd Pass, Code Analysis Pass, AST Creation, AST Clients, Function Analysis, FPA Function Analysis, EspX CFG Build, EspX Function Analysis, EspX Path-sensitive Analysis 
+  6139105, 1184276, 57333281, 382762, 48323001, 46771695, 4680989, 3837409, 36653783, 34256891 
+Compiler Passes [percentages are "percentage of parent" ("percentage of total")]: 
+  Front End Pass = 9.49% 
+  Back End Pass = 1.83% 
+  Code Analysis Pass = 88.67% 
+    Compilation + Miscellaneous = 15.05% (13.34%) 
+    AST Creation = 0.67% (0.59%) 
+    All AST Clients = 84.28% (74.74%) 
+      Miscellaneous = 3.21% (2.40%) 
+      Function Analysis = 96.79% (72.34%) 
+        Miscellaneous = 3.42% (2.47%) 
+        PREfast's Function Path Analysis = 10.01% (7.24%) 
+        EspX CFG Building = 8.20% (5.94%) 
+        EspX All Analysis = 78.37% (56.69%) 
+          Path-sensitive Analysis = 93.46% (52.98%) 
+          Data-flow Analysis + Miscellaneous = 6.54% (3.71%) 
+Number of TUs with long Code Analysis Pass compared to Front End Pass: 
+  600% or more: 1837 (55.85%) 
+  300% or more: 945 (28.73%) 
+  150% or more: 418 (12.71%) 
+  Less than 150%: 89 (2.71%) 
+```
+
+### Per translation unit Performance Data in CSV Format
+
+Here is an example of the per translation performance data output. First row is the colum header, and the rest are per translation unit performance data - i.e., execution times of each step in milliseconds.
+
+This can be imported into tools like MS Excel to create various reports.
+
+File Path, Front End Pass, BackEnd Pass, Code Analysis Pass, AST Creation, AST Clients, Function Analysis, FPA Function Analysis, EspX CFG Build, EspX Function Analysis, EspX Path-sensitive Analysis
+d:\test\file1.cpp, 253976, 111859, 5077991, 14019, 249835, 229841, 84557, 0, 0, 0
+d:\test\file2.cpp, 0, 0, 108231, 56, 467, 312, 123, 0, 0, 0
```

Comment:
**suggestion (readability)**: Wrap this CSV snippet in triple-ticks.

It is difficult to read, and the newlines do not render properly.